app/main.py

```
@@ -324,7 +324,45 @@ async def download_event_ics(event_id: str, client: httpx.AsyncClient = Depends(
             "Content-Disposition": f'attachment; filename="{filename}"'},
     )
 
-# Health check endpoint
+# Fetch all copyright info for images from the CMS
+
+
+@app.get("/copyrights/", description="Get copyrights for images", tags=["copyright"])
+async def get_all_copyright_info(limit: int = 30, client: httpx.AsyncClient = Depends(_get_cms_client)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)